lte/gateway/c/core/oai/lib/3gpp/3gpp_24.008.h

```diff
@@ -886,6 +886,11 @@ int decode_access_point_name_ie(
 /* Both directions:*/
 #define PCO_CI_IM_CN_SUBSYSTEM_SIGNALING_FLAG (0x0002)
 
+/* IPCP Macros */
+#define IPCP_CODE_BYTES 1
+#define IPCP_IDENTIFIER_BYTES 1
+#define IPC_LENGTH_BYTES 2
+
```

Comment:
Not sure IPCP_* should go into 24.008 related .h file.

Reply:
Comments addressed.